common/djangoapps/third_party_auth/decorators.py

```diff
@@ -30,3 +34,40 @@ def wrapped_view(request, *args, **kwargs):
         resp['X-Frame-Options'] = x_frame_option
         return resp
     return wraps(view_func, assigned=available_attrs(view_func))(wrapped_view)
+
+
+from urllib import urlencode
```

Comment:
Yes, it should. It should also be from `six`. All this will be done prior to merge.